components/main/GoalMenuContainer/GoalMenuContainer.tsx

```
@@ -0,0 +1,75 @@
+import ModalAddingGoal from "@/components/shared/Modal/ModalAddingGoal/ModalAddingGoal";
+import { createNewGoal } from "@/lib/main/goalFetching";
+
+type GoalMenuInfo = Pick<GoalMenuProps, "goal" | "percentage"> & {
+  goalId: string;
+};
+interface GoalMenuContainerProps {
+  initGoalsInfo?: GoalMenuInfo[];
+  initSelectedGoalInfo?: GoalMenuProps["goal"];
+}
+
+const GoalMenuContainer = ({}: GoalMenuContainerProps) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)